src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1584,7 +1597,7 @@ struct activation_random_test : testing::TestWithParam<activation_random_test_pa
                     for (size_t xi = 0; xi < x; ++xi) {
                         auto ref_out_val = ref_ptr[ref_out_offset + xi * ref_x_pitch];
                         auto opt_out_val = opt_ptr[opt_out_offset + xi * opt_x_pitch];
-                        EXPECT_EQ(ref_out_val, opt_out_val);
+                        EXPECT_TRUE(compare_output(ref_out_val, opt_out_val));
```

Comment:
I think you can use `EXPECT_NEAR`, instead.

Reply:
Done.